functions.php

```diff
@@ -44,10 +44,13 @@ function ampconf_setup() {
 		 */
 		add_theme_support( 'post-thumbnails' );
 
-		// This theme uses wp_nav_menu() in one location.
+		/***
```

Comment:
Trivial but `/**` is reserved for phpdoc, like for functions, methods, variable definitions, etc. For other cases, just use:

```
/*
...
*/
```

But since this is a single-line comment anyway, then `//` I think is right.

Reply:
Comment changed.